### PR TITLE
Bump up runtime-spec dependency to v1.0.0

### PR DESCRIPTION
**Pull request by mrunalp**
@opencontainers/runtime-tools-maintainers PTAL
Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

**Comment by vbatts** [comment]
LGTM

File: generate/generate.go [hunk 1]
```diff
@@ -604,6 +598,12 @@ func (g *Generator) SetLinuxResourcesMemorySwappiness(swappiness uint64) {
 	g.spec.Linux.Resources.Memory.Swappiness = &swappiness
 }
 
+// SetLinuxResourcesDisableOOMKiller sets g.spec.Linux.Resources.Memory.DisableOOMKiller.
+func (g *Generator) SetLinuxResourcesDisableOOMKiller(disable bool) {
```
Comment (wking) [inline generate/generate.go]: Do you want to use `SetLinuxResourcesMemoryDisableOOMKiller` to match [these][1]?

[1]: https://github.com/opencontainers/runtime-tools/blob/6210a304b5ef728abbd40f0d6ccf61e1a83e44fc/generate/generate.go#L571-L599

Comment (mrunalp) [inline generate/generate.go]: done

**Comment by vbatts** [comment]
LGTM

**Comment by hqhq** [comment]
LGTM

**Pull request merged**